lib/core/versions/0.9.0/CreateOperation.ts

```diff
@@ -11,6 +11,7 @@ import SidetreeError from '../../../common/SidetreeError';
 interface SuffixDataModel {
   deltaHash: string;
   recoveryCommitment: string;
+  type?: string;
```

Comment:
why is type optional?

Reply:
It is an optional field to denote what type of DID it is, a person, a repo, a dog ...etc